app/src/components/base/DetailCard/DetailCard.test.tsx

```diff
@@ -10,7 +10,7 @@ const mockRows = [
   { key: 5, detailKey: 'Detail Key 5', value: 'Value 5' },
 ];
 
-describe(DetailCard, () => {
+describe.skip(DetailCard, () => {
```

Comment:
Do we need to skip this?

Reply:
Yeah.  Unfortunately, the test is going to need to be updated

Reply:
https://github.com/casper-network/casper-blockexplorer-frontend/issues/293